ci-operator/templates/openshift/openshift-azure/build-node-image.yaml

```diff
@@ -25,6 +25,8 @@ parameters:
 - name: DEPLOY_OS
   value: "rhel7"
   required: true
+- name: RUNNING_UNDER_TEST
```

Comment:
@kargakis is this sufficient to allow a parameter to be passed?

Reply:
Why are you adding it in the node build job?